python/ray/util/accelerators/intel-gpu.md

```diff
@@ -0,0 +1,75 @@
+
+[oneAPI](https://en.wikipedia.org/wiki/OneAPI_(compute_acceleration)) is an open standard adopted by Intel for a unified programming model and APIs to deliver a common developer experience across accelerator architectures.
+
+Intel XPU is programmed with Data Parallel C++ (DPC++) based on open [SYCL](https://en.wikipedia.org/wiki/SYCL) standard. Check [here](https://dgpu-docs.intel.com/_images/one_api_sw_stack.png) for overall Intel oneAPI software stack.
+
+Intel XPU support has been added for some most popular machine learning projects (E.g. [Intel® Extension for PyTorch](https://www.intel.cn/content/www/cn/zh/developer/articles/technical/introducing-intel-extension-for-pytorch-for-gpus.html) and [Intel® Extension for DeepSpeed](https://github.com/intel/intel-extension-for-deepspeed)).
+
+Intel GPU is one of the XPU categories. Check official [Intel Graphics Processor Table](https://dgpu-docs.intel.com/devices/hardware-table.html) for supported devices.
+
+Intel GPU can be detected by environment variable [ONEAPI_DEVICE_SELECTOR](https://intel.github.io/llvm-docs/EnvironmentVariables.html#oneapi-device-selector).
+The device selection environment variable can be used to limit the choice of devices available when the SYCL-using application is run. Useful for limiting devices to a certain type (like GPUs or accelerators) or backends (like Level Zero or OpenCL).
+
+## How it works with Intel GPUs
+
+We have expand the current GPUs type support. Ray cluster can use different vendor's GPU.
+
+### Detection
+Python module libray [dpctl](https://github.com/IntelPython/dpctl) is imported to detect XPU device.
+To detect Intel GPU device, `ONEAPI_DEVICE_SELECTOR` should provide `backend` and  `device_type`
+Such as:
+``` bash
+# select all level_zero gpu device under XPU
+ONEAPI_DEVICE_SELECTOR="level_zero:gpu"
+
+# select level_zero device [0,1,2]
+ONEAPI_DEVICE_SELECTOR="level_zero:0,1,2"
+```
+
+### Use Intel GPU in Ray
+We have upgraded Ray to allow users to use Intel GPUs almost seamlessly, just like Nvidia’s different GPU devices.
+
+``` python
+from ray.util.accelerators import NVIDIA_TESLA_V100, INTEL_MAX_1550
+
+# add a node with Nvidia GPU to cluster
+cluster.add_node(num_cpus=1, num_gpus=8, resources={f"acceleator_type:{NVIDIA_TESLA_V100}": 1})
+
+# add a node with Intel GPU to cluster
+cluster.add_node(num_cpus=1, num_gpus=8, resources={f"acceleator_type:{INTEL_MAX_1550}": 1})
+
+ray.init(address=cluster.address)
+
+# use Nvidia GPU to train
+@ray.remote(num_gpus=1, accelerator_type=NVIDIA_TESLA_V100)
+def train(data):
+    return "This function was run on a node with a Nvidia Tesla V100 GPU"
+
+# use Intel GPU to infer
+ray.get(train.remote(1))
+
+@ray.remote(num_gpus=1, accelerator_type=INTEL_MAX_1550)
+def infer(data):
+    return "This function was run on a node with an Intel Max 1550 GPU"
+
+ray.get(infer.remote(1))
```

Comment:
as example, could you remove "data" and "1" if it's not used inside the function.